src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/filepage/FilePage.java

```diff
@@ -96,8 +96,19 @@ public void verifyEmbeddedVideoIsPresent() {
         true);
   }
 
+  public boolean isNoFileTextBoxVisible(){
+    try{
+      wait.forElementVisible(noFileTextBox);
+      PageObjectLogging.log("isNoFileTextBoxVisible", "No-file textbox is visible ", true);
+      return true;
+    } catch (TimeoutException e){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)